src/components/interaction-designer/Block.vue

```diff
@@ -20,7 +16,7 @@
         'source-block-having-active-connection': isAssociatedWithActiveConnectionAsSourceBlock,
         'target-block-having-active-connection': isAssociatedWithActiveConnectionAsTargetBlock,
         'target-block-waiting-for-connection': isWaitingForConnection,
-        'has-toolbar': isBlockSelected || shouldShowBlockEditor,
+        'has-toolbar': isBlockSelected || shouldShowBlockEditorForCurrentBlock,
```

Comment:
renamed it to avoid confusion with something like `isBlockEditorOpen` from the store